docs/platforms/javascript/guides/react/features/component-tracking.mdx

```diff
@@ -4,6 +4,84 @@ excerpt: ""
 description: "Learn how Sentry's React SDK allows you to monitor the rendering performance of your application and its components."
 ---
 
+Sentry offers the ability to track your React components to unlock additional insights in your application.
+
+## Component Name Tracking
+
+You can track the names of React components in your application via a [Babel plugin](https://www.npmjs.com/package/@sentry/babel-plugin-component-annotate), which can unlock powerful workflows. For example, you can search for replays where a certain React component was clicked on. You can also see the name of the component that was clicked on in breadcrumbs, and spans in Performance. This considerably decreases the level of ambiguity regarding the specific element on the page that was interacted with, as you would normally see just the element's CSS selector (which is also subject to further ambiguity during the minification process).
+
+There will be more features released down the line that will leverage component name tracking, so it is highly recommended to configure your project to use it.
+
+### How to Install
+
+<Alert>
+
+Please note that your Sentry browser SDK must be at version `7.91.0` or higher before you can use these features.
+Only React components in `.jsx` or `.tsx` files can be tracked.
+
+</Alert>
+
+There are two different ways you can setup React component name tracking in your application:
+
+1. Install [Sentry's bundler plugins](https://www.npmjs.com/package/@sentry/bundler-plugin-core)
+2. Directly install [@sentry/babel-plugin-component-annotate](https://www.npmjs.com/package/@sentry/babel-plugin-component-annotate)
+
+It's recommended that you use the first option, as the Sentry bundler plugins come packed with additional useful features to enrich your Sentry workflows, such as automatic sourcemap upload,
+release creation, automatic release name discovery, and release injection. However, you have the option of installing the standalone Babel plugin as well.
```

Comment:
```suggestion
We recommend that you use the first option because the Sentry bundler plugins come packed with additional useful features to enrich your Sentry workflows, including: automatic sourcemap upload,
release creation, automatic release name discovery, and release injection. 
```